includes/admin/post-transition.php

```diff
@@ -92,7 +94,7 @@ function( $post ) {
  * @param int  $post_id The current post ID.
  * @param bool $force   Publish tweet regardless of autoshare enabled or disabled on post.
  *
- * @return object
+ * @return bool|void
```

Comment:
@iamdharmesh I think we should strictly set it to `bool`.

In case this function returns `void` which will evaluate to `null`, that `null` will be set here:

https://github.com/10up/autoshare-for-twitter/blob/73ab6c80061ac3862a16909a9c98ebfee71d836a/includes/admin/post-transition.php#L171

In JS, we strictly check it against `false` here:

https://github.com/10up/autoshare-for-twitter/blob/73ab6c80061ac3862a16909a9c98ebfee71d836a/assets/js/admin-autoshare-for-twitter-classic-editor.js#L218

What do you think?

Reply:
Yeah, not related to this PR but important enough to fix it here. Updated. Thanks